### PR TITLE
Dps tutorial v1

### PR DESCRIPTION
**Pull request by rtapella**
Initial version of the DPS in-depth tutorial:
- example DPS tutorial repository
- cover all the steps from writing your algorithm to running it in DPS
- initial version covers the GUI-based registration and execution

### CODE REVIEW

File: docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,559 @@
+{
```
Comment (omshinde) [inline docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb:1]: <ul><li>sets the correct python environment for your code to run</li><li>the best way to execute your algorithm with a custom environment is to use <code>conda run</code>, as shown in this script(<code>conda run --live-stream --name dps_tutorial python ${basedir}/gdal_wrapper.py --input_file ${INPUT_FILENAME} --output_file output/${OUTPUT_FILENAME} --outsize ${REDUCTION_SIZE}</code></li><li> )</li></ul><p> </p><blockquote> )</blockquote><p>Do we need this <code>)</code> to be a separate bullet point?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (omshinde) [inline docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb:1]: <p>Probably a minor formatting suggestion but it looks nice to start a sentence with Uppercase.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/389/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Comment by omshinde** [comment]
This looks good to me. I suggested some minor changes. Also, it seems that there's a TODO section towards the end of the notebook. Otherwise, nice work.

File: docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb (hunk 1 above)
Comment (rtapella) [reply docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb:1]: <p>Hm I do not see that extra bullet point in my browser...</p><p> </p>

Comment (rtapella) [reply docs/source/technical_tutorials/dps_tutorial/dps_tutorial_demo.ipynb:1]: <p>pushed a fix for uppercasing those bullets</p>

**Comment by rtapella** [comment]
also removed the TODO section
please check again @omshinde

**Review by omshinde — APPROVED** [review]
Thank you @rtapella .. The revisions look good to me.

**Comment by rtapella** [comment]
resolves https://github.com/MAAP-Project/maap-documentation/issues/278

**Pull request merged**